test/server.spec.js

```diff
@@ -48,21 +47,6 @@ describe('Subgenerator server of quarkus JHipster blueprint', () => {
         it('contains hibernate second level cache needle', () => {
             assert.fileContent(`${SERVER_MAIN_RES_DIR}application.properties`, '# jhipster-quarkus-needle-hibernate-cache-add-entry');
         });
-
-        it('contains h2 servlet data in web.xml', () => {
```

Comment:
@avdev4j  Do you remember why we needed the web.xml file?
It seems that the h2 console is not available.